chain/ethereum/src/block_stream.rs

```diff
@@ -12,19 +12,18 @@ use graph::prelude::{
 };
 use tokio::timer::Delay;
 
-const FAST_SCAN_SPEEDUP: u64 = 10;
-
 lazy_static! {
-    /// Number of blocks to request in each chunk.
-    static ref ETHEREUM_BLOCK_RANGE_SIZE: u64 = ::std::env::var("ETHEREUM_BLOCK_RANGE_SIZE")
-        .unwrap_or("10000".into())
+    /// Maximum number of blocks to request in each chunk.
+    static ref MAX_BLOCK_RANGE_SIZE: u64 = std::env::var("GRAPH_ETHEREUM_MAX_BLOCK_RANGE_SIZE")
```

Comment:
Should we keep this default in production? Right now the existing env var in community is set to `3000` and `10000` for customers

Reply:
The `MAX_BLOCK_RANGE_SIZE` is essentially to prevent us from scanning the entire chain at once. Scanning 100k blocks at once may cause us to go way over the target in bad cases, but that will happen only once and then the range will adapt.

From my testing I believe these defaults should work out fine and we won't need to tinker with these variables, but if they turn out to be too aggresive we can always lower as a short term solution.

Reply:
Won't reducing 100k to 10k have a negative performance impact on early block scanning of most subgraphs?